### PR TITLE
Clean up 1.22 jobs for sig-windows

### PR DESCRIPTION
**Pull request by jsturtevant**
The 1.22 branch and release jobs were generated last night.  This does some slight clean up:

- the containerd nightly job doesn't need to be in release monitoring, remove and remove annotation so it is not forked next time
- moves the 1.22 job to sig-windows with rest of job configs
- adds a 1.22 tab for sig-windows dashboard
- removed 1.18 jobs

/sig windows
/sig release

/cc @marosset @chewong @ravisantoshgudimetla

### CODE REVIEW

**Review by chewong — COMMENTED** [review]
Let's remove 1.18 dashboards in this PR (https://testgrid.k8s.io/sig-release-master-blocking has stopped tracking 1.18 as well)

File: config/jobs/kubernetes-sigs/sig-windows/release-1.22-windows.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - --aksengine-win-binaries
+      - --aksengine-deploy-custom-k8s
+      - --test_args=--node-os-distro=windows -docker-config-file=$(DOCKER_CONFIG_FILE)
+        --ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]|\[sig-apps\].CronJob|\[sig-api-machinery\].ResourceQuota|\[sig-network\].EndpointSlice
+        --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Slow\]|\[alpha\]|GMSA|Guestbook.application.should.create.and.stop.a.working.application|device.plugin.for.Windows
+      - --ginkgo-parallel=4
+      command:
+      - runner.sh
+      - kubetest
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20210721-2b77449-1.22
```
Comment (jsturtevant) [inline config/jobs/kubernetes-sigs/sig-windows/release-1.22-windows.yaml:52]: I made sure we have the right kubekins for 1.22 from this fix: https://github.com/kubernetes/test-infra/pull/22990

**Comment by puerco** [comment]
Thank you @jsturtevant !
/approve

**Review by ravisantoshgudimetla — APPROVED** [review]
/lgtm

Thank you for working on this @jsturtevant

**Pull request merged**